security/security-core/src/main/java/io/camunda/security/auth/OidcPrincipalLoader.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.security.auth;
+
+import com.jayway.jsonpath.Configuration;
+import com.jayway.jsonpath.JsonPath;
+import com.jayway.jsonpath.JsonPathException;
+import com.jayway.jsonpath.Option;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class OidcPrincipalLoader {
+  public static final Configuration CONFIGURATION =
```

Comment:
:question: Does this need to be public?

Reply:
No, fixed it in b39efed99a2d0f85dde096eb85dc256eeaa9476b :+1: